web/crux-ui/locales/en/registries.json

```diff
@@ -29,8 +29,8 @@
 
   "type": {
     "v2": "V2 Registry",
-    "hub": "Docker HUB",
-    "gitlab": "Gitlab Registry",
+    "hub": "Docker Hub",
+    "gitlab": "GitLab Registry",
```

Comment:
For the sake of correctness, the real name of that software is **GitLab Container Registry**. Shouldn't we use their _official_ name?

Reply:
I guess it's too long @polaroi8d ?